src/Vendor/Order/OrderService.php

```diff
@@ -278,4 +278,24 @@ public function generateMondialRelayLabel(int $orderId, CreateLabelCommand $comm
 
         return new MondialRelayLabel($result);
     }
+
+    /**
+     * @param int $orderId
+     * @param string $details
+     * @return OrderService
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     * @throws \Wizaplace\SDK\Authentication\AuthenticationRequired
+     * @throws \Wizaplace\SDK\Exception\JsonDecodingError
+     */
+    public function setOrderDetails(int $orderId, string $details): self
+    {
+        $this->client->mustBeAuthenticated();
+        $this->client->post("orders/${orderId}/details", [
```

Comment:
```suggestion
        $this->client->post("orders/$orderId/details", [
```
Elles servaient à quoi ces accolades ?

Reply:
C'est une autre manière de concaténer les chaînes.

Reply:
Dans ce cas là les accolades ne sont pas utile